sample-tracking-status-overview-app/src/main/groovy/life/qbic/portal/sampletracking/components/projectoverview/ProjectOverviewView.groovy

```diff
@@ -256,14 +263,16 @@ class ProjectOverviewView extends VerticalLayout{
 
     private void tryToDownloadManifest() {
         try {
-            Optional.ofNullable(viewModel.selectedProject).filter({it.sampleDataAvailable > 0 }).ifPresent({
+            Optional.ofNullable(viewModel.selectedProject).filter({it.sampleDataAvailable.passingSamples > 0 }).ifPresent({
```

Comment:
To improve logging you could do the following
```suggestion
            Optional<ProjectSummary> selectedSummary = Optional.ofNullable(viewModel.selectedProject)
            selectedSummary.filter({it.sampleDataAvailable.passingSamples > 0 }).ifPresent({
```
and use `selectedSummary` in your logging.